### PR TITLE
fix: footer entity logo display

### PR DESCRIPTION
**Pull request by sandroscosta**
fix logo display on footer funding section

### CODE REVIEW

**Comment by sandroscosta** [comment]
Closes https://github.com/fccn/nau-technical/issues/97

File: edx-platform/nau-basic/lms/static/sass/partials/lms/theme/_footer.scss [hunk 1]
```diff
@@ -105,12 +105,7 @@ footer {
             padding-left: 5px;            
 
             img {
```
Comment (igobranco) [inline edx-platform/nau-basic/lms/static/sass/partials/lms/theme/_footer.scss:107]: Add:

```scss
img {
    max-width: calc(100vw - 20px);
    max-height: 4.6rem;
}
```

**Review by igobranco — CHANGES_REQUESTED** [review]
It brakes on mobile:
image

**Review by igobranco — APPROVED** [review]
LGTM ✔️

**Pull request merged**